functions/bot_moderator_function.py

```diff
@@ -118,6 +119,7 @@ def main(event):
                 limit=2))
             )
         
+        response = response.get_result()
```

Comment:
interesting... This isn't a python version issue.  It is a watson SDK issue, I think.  Looks to me like the 3.6 is using the older SDK.  And the 3.7 has the newer one which broke stuff and requires .get_result()

I don't think you need to explain any the SDK issue (your way is fine for now), but heads-up because they'll probably update the 3.6 one eventually, right?